paddle/platform/device_context.h

```diff
@@ -74,8 +75,7 @@ class CUDADeviceContext : public DeviceContext {
  public:
   explicit CUDADeviceContext(const GPUPlace gpu_place) : gpu_place_(gpu_place) {
     GPUPlaceGuard guard(gpu_place_);
-    paddle::platform::throw_on_error(cudaStreamCreate(&stream_),
-                                     "cudaStreamCreate failed");
+    PADDLE_ENFORCE(cudaStreamCreate(&stream_), "cudaStreamCreate failed");
```

Comment:
In my opinion, `PADDLE_ENFORCE(condition, ...)` means `condition` must execute correctly. No matter the return of condition is 0 or 1, the internal of `PADDLE_ENFORCE` already figured out how to deal with it.